### ISSUE TITLE
Generate Tiled Regions Feedback

### ISSUE THREAD

**Issue by srivarra** [angelolab/toffy#95]
**Is your feature request related to a problem? Please describe.**

Applying feedback from those who tested the *Generate Tiled Regions* notebook.

**Describe the solution you'd like**

## Clarifying details to add

1. Briefly explain **where** the *JSON* file is located, **what** it is, and **how** we will be using it.
   1. For example, brief description on it's contents.
   2. State that it's a file generated by a FOV on the MIBI, not something that the users need to create themselves.
2. Give explicit instructions on the directory structure and required filenames.
3. The *Base* directory is ambiguous on the TeamViewer Windows PC.
   1. They recommended creating a directory for users to organize all JSON files needed.
4. `moly_path = os.path.join(json_tiling_dir, 'moly_point.json')` Is currently unused. Will it be needed in the future?
5. The term *user prompt* is a bit technical. Rephrase this to be intuitive to understand for those with no technical background.
6. Add `print` statements or a `tqdm` progress bar to show that it is indeed working, just taking a second or two.
7. Fix the duplicate FOV issue when an FOV is imported.
   1. Either request the user to delete the duplicate FOV themselves, or run this automatically.

**Describe alternatives you've considered**

**Additional context**

**Comment by ngreenwald** [comment]
bullet 3 is getting resolved in one of my open PRs. 
Bullet 4 has an issue open for it. You can tackle it if you want
bullet 7 is resolved now

**Comment by ngreenwald** [comment]
@angelolab/computational_team Any of this still need to be addressed?
